_posts/2024-09-01-openloco-v24.09.md

```diff
@@ -0,0 +1,84 @@
+the road paint functions represent a huge chunk ~3% of the vanilla game. Like the track paint
+functions they are repetitive and probably were automatically generated by some sort of code generator.
+Annoyingly they don't share enough to use the same code as we did for track paint but we could use
+the same technique (TestPaint) to generate our C++ version. With them implemented its another big
+jump in percentage completed and another step towards being able to add new road pieces.
+
+## Implement Create Airport Game Command (#2585, #2568, #2598, #2603)
+
+On the implementing game commands project this month we finished off airport creation and track
+removal. Airport creation a simple function that is very similar to the other building related 
+game commands (Industry, Building, Docks) but it identified a slew of small mistakes we had made
+in previously merged code. Issues:
+
+  1. Sounds generated using the wrong random number generator. Turned out that there is one random
+	 number generator used only for sounds and ui and another for game logic. We had accidentally
+	 been using the game logic one for sounds. It doesn't really change anything but it would
+	 have caused issues when it came to multiplayer.
+  2. Large square stations couldn't be created. We had known for a few releases that our code for
+	 finding nearby stations wasn't quite correct and would prevent 16x16 square stations. It
+	 wasn't until Create Airport though until we had to solve that issue as it was causing a very
+	 minor desync.
+  3. Road vehicles may use the wrong track type when reversing. This is actually a vanilla mistake
+	 which we accidentally duplicated. Its unlikely to cause any real difference as it appears this
+     was only being looked at for working out max speeds on turns. But it is possible that this
+	 might cause a crash. Eitherway its now corrected. It might seem odd that we found this whilst
+	 doing an airport function but its because in the process of resolving the nearby station issue
+	 I implemented another function that was also used by road station placement which surfaced this
+	 bug.
+
+## Implement Track Removal Game Command (#2571)
+
+@AaronVanGeffen implemented the track removal game command this month. Much more straight forward
+compared to track creation but still complex in its own ways. With this one implemented we now have
+only 14/84 left to implement. Four of which are road versions of track commands, four that look to
+be Ai only functions and the rest that are multiplayer related and might be able to be ignored.
+
+## Implement Map Generator Terrain In Random Areas (#2594)
+
+Another two functions from @AaronVanGeffen that finished off the last of the map generator! We
+now have complete control over map generation and @AaronVanGeffen has been using these new powers
+to start adding new features to the map generator (e.g. rivers, [ADD MORE]...).
```

Comment:
was the `ADD MORE` supposed to be filled in, or is it like a "we'll add more in this space in future"

Reply:
It was for @AaronVanGeffen to fill in with anything else